### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by chuanqizhu0820**
## Summary
- clicks (or taps) the button to check project details => the popup with details about the project appears.
- user clicks (or taps) the close (X) button => the popup disappears.
- stick to the design of the template 5 in Figma.
- use a JavaScript array to store all of the information for all projects.
- update the main page so that the projects section is created dynamically using the information stored in that JavaScript object.
- pair-programming for this project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,7 +160,7 @@ <h3 class="work-name">Multi-Post Stories</h3>
               <li class="gird">Bootstrap</li>
               <li class="grid0">Ruby</li>
             </ul>
-            <button>See Project</button>
+            <button class="btn-window" id="work1">See Project</button>
           </div>
         </div>
         <div class="work2 flex">
```
Comment (hemantso) [inline index.html]: - [x] As per project requirement, The projects section is created dynamically with the information from the projects array. Let's try to fix this issue by removing already created HTML code for the projects list and instead it should be added via JavaScript.

image

File: main.js [hunk 2]
```diff
@@ -21,4 +21,80 @@ closebutton.addEventListener('click', closeMenu);
...
+  const workid = this.id;
+
+  const protitle = document.querySelector('#windowtitle');
+  protitle.textContent = projectInfo[workid].name;
+
+  const proimg = document.querySelector('#windowimg');
+  proimg.src = projectInfo[workid].image;
+
+  const protech = document.querySelectorAll('#windowtech li');
+  protech.forEach((item, i) => {
```
Comment (hemantso) [inline main.js]: - [x]  As per Figma design the project title and popup close icon should be on extreme end of the popup. Kindly check the screenshot below and try to make the required changes for both mobile views as well as desktop views.

- [x] On the Desktop view, The project description should be in the center. Kindly compare the below image and fix the issue.

Yours
image


Figma Design
image

Your pop up on mobile view
image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly try to add information about running the project locally and the acknowledgment section in your README file. 

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by barackm — CHANGES_REQUESTED** [review]
Hi team, 👋 

Great work on making the changes requested by a previous reviewer 👏 👏👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 😉  😉 

 # STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 

- [x] It seems that you haven't yet implemented the requested changes by the previous reviewer, since there is any commit available after the review, Kindly let's implement the requested changes by the previous reviewer 😉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @chuanqizhu0820, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -101,100 +132,92 @@ <h3>Tools</h3>
...
+                              dummy text.
+                            </p>
+                            <ul class="work-skills grid-container">
+                              <li class="gird">CSS</li>
+                              <li class="gird">HTML</li>
+                              <li class="gird">Bootstrap</li>
+                              <li class="grid0">Ruby</li>
+                            </ul>
+                            <button class="btn-window" id="work1">See Project</button>
+                          </div>
```
Comment (mke2111) [inline index.html]: - [x] Clean up this file. It is a better practice to remove pieces of commented-out code from you repo.

File: main.js [hunk 4]
```diff
@@ -21,4 +21,146 @@ closebutton.addEventListener('click', closeMenu);
...
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+};
+
+const workcontainer = document.querySelector('.work-container');
+
+let projhtml = '';
+
+projectInfo.forEach((item) => {
```
Comment (mke2111) [inline main.js]: - [ ] The given design specs from figma have not been met. Please ensure that the popup window match the given specs.
 
Screenshot from 2021-09-03 14-06-36

Comment (chuanqizhu0820) [reply main.js]: Hi, the popup window does not look like this actually. Could you check it once again? I have already made changes according to the first reviewer's suggestions.

File: main.js (hunk 2 above)
Comment (chuanqizhu0820) [reply main.js]: Hello, @hemant-soni-vst-au4. I made the changes according to your suggestions, but the second reviewer does not approve it. What should I do?

**Comment by mimipeshy** [comment]
## Hello @chuanqizhu0820  👋,

_This is an additional review as a result of the second opinion request._

From the project description, you are only supposed to work on the position of the image and the description switch places (picture is on the left and description on the right for the first project, description on the left, and picture on the right for the second project and so on and so forth). This has already been done on your part. The pop-up window will be an OPTION at this point, you can work on it if you wish to

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take few hours!

Keep rocking 🚀!

**Pull request merged**